### PR TITLE
Fixes issue related to non-multi-size FF ads not centering correctly

### PR DESCRIPTION
**Pull request by glevitzky**
The issue is fixed by using the slot's width/height attributes to size the iframe instead of defaulting to measuring the available space.

### CODE REVIEW

File: extensions/amp-ad-network-doubleclick-impl/0.1/test/test-amp-ad-network-doubleclick-impl.js [hunk 1]
```diff
@@ -1408,4 +1408,144 @@ describes.sandboxed('amp-ad-network-doubleclick-impl', {}, () => {
...
+          'type': 'doubleclick',
+          'layout': 'fixed',
+        });
+        doc.body.appendChild(element);
+
+        impl = new AmpAdNetworkDoubleclickImpl(element);
+        impl.initialSize_ = {width: 200, height: 50};
+
+        // Boilerplate stubbing
+        //sandbox.stub(doc, 'readyState', 'complete');
```
Comment (taymonbeal) [inline extensions/amp-ad-network-doubleclick-impl/0.1/test/test-amp-ad-network-doubleclick-impl.js]: What's this doing here?

Comment (glevitzky) [inline extensions/amp-ad-network-doubleclick-impl/0.1/test/test-amp-ad-network-doubleclick-impl.js]: Removed.

File: extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js [hunk 2]
```diff
@@ -529,7 +539,8 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
     // Force size of frame to match creative or, if creative size is unknown,
     // the slot. This ensures that the creative is centered in the former case,
     // and not truncated in the latter.
-    const size = this.returnedSize_ || this.getIntersectionElementLayoutBox();
+    // TODO(levitzky) Figure out the behavior of responsive + multi-size.
```
Comment (keithwrightbos) [inline extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js]: Should we warn?  Log how often it happens?

Comment (glevitzky) [inline extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js]: Added warning. Not sure how I would log this for our records.

**Pull request merged**